lib/LessParser.js

```diff
@@ -67,9 +67,8 @@ module.exports = class LessParser extends Parser {
     const node = new Comment();
     const text = token[1].slice(2);
 
-    this.init(node, token[2], token[3]);
-
-    node.source.end = { line: token[4], column: token[5] };
+    this.init(node, token[2]);
+    node.source.end = this.getPosition(token[3] || token[2]);
```

Comment:
for these changes here and in inline-comment.js, could you comment with what the old `newToken` values were, how they were incorrect, and the shape of the new `newToken`? I'd just like it documented for reference in the future.

Reply:
Sure, found [the related PR](https://github.com/postcss/postcss/pull/1326) in postcss.

- Token definition used to be `[type, content, start_line, start_column, end_line, end_column]` with last two values being optional. It has been changed to `[type, content, start_byte, end_byte]` with the last value being optional.
Ref: https://cultofmartians.com/tasks/postcss-offset.html#task (attached to the postcss PR)
And: https://github.com/postcss/postcss/blob/v7/lib/tokenize.es6 source code.

- Same story for the `Parser::init` function, which had the following arguments : `node, line, column`; but have been changed to : `node, offset`.
In V7: https://github.com/postcss/postcss/blob/67e3d7b3402c5d3d036ab7c1e781f86910d6ca72/lib/parser.es6#L352
In V8: https://github.com/postcss/postcss/blob/1545d8aabd2a920c76a70fdd4e7c43a9464fba5e/lib/parser.js#L387

- And finally, Parser class now provides a `Parser::getPosition` function which takes an offset as argument and returns a node source location object (instead of having to forge it by hand with line/column/offset properties).
Ref: https://github.com/postcss/postcss/blob/1545d8aabd2a920c76a70fdd4e7c43a9464fba5e/lib/parser.js#L378

Let me know if you need anything else.

Reply:
That's great, thank you.